### PR TITLE
New DotNetCore Sdks 2024-03-13

### PR DESCRIPTION
**Pull request by harryli0108**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: build/constants.yaml [hunk 1]
```diff
@@ -47,29 +47,29 @@
...
+    - 3.1-debian-bullseye
+    - 3.1-debian-buster
+    - 5.0-debian-buster
+    - 6.0-debian-bullseye
+    - 6.0-debian-buster
+    - 7.0-debian-bullseye
+    - 7.0-debian-buster
+    - 8.0-debian-bullseye
+    - 8.0-debian-bookworm
+    - dynamic-debian-buster
```
Comment (sarsharma) [inline build/constants.yaml:62]: Let's get clarity on these dynamic images also for the future

File: build/constants.yaml [hunk 2]
```diff
@@ -160,29 +160,29 @@
...
+    - 3.9-debian-bullseye
+    - 3.9-debian-buster
+    - 3.10-debian-bookworm
+    - 3.10-debian-bullseye
+    - 3.10-debian-buster
+    - 3.11-debian-bookworm
+    - 3.11-debian-bullseye
+    - 3.12-debian-bookworm
+    - 3.12-debian-bullseye
+    - dynamic-debian-buster
```
Comment (sarsharma) [inline build/constants.yaml:178]: any idea on why this automation keeps changing the formatting with each PR?

File: build/constants.yaml (hunk 1 above)
Comment (kumaraksh1) [reply build/constants.yaml:62]: Sure sure..

File: doc/supportedPlatformVersions.md [hunk 3]
```diff
@@ -20,8 +20,10 @@
 - 8.0.100-rc.1.23463.5
 - 8.0.100-rc.2.23502.2
 - 8.0.100
+- 8.0.100
```
[the remark this answers is not in the document]
Comment (kumaraksh1) [reply doc/supportedPlatformVersions.md:23]: I saw but its just a readme so lest as is.

File: build/constants.yaml (hunk 2 above)
Comment (kumaraksh1) [reply build/constants.yaml:178]: It's like automation workflow indent the content of the file.
From next PR, i think it wont be like this.

File: doc/supportedPlatformVersions.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kumaraksh1) [reply doc/supportedPlatformVersions.md]: updated btw

**Review by sarsharma — APPROVED** [review]

**Pull request merged**